conf/routes

```diff
@@ -105,6 +105,13 @@ POST   /v3/backfill/:name/start    @org.sagebionetworks.bridge.play.controllers.
 GET    /v3/cache           @org.sagebionetworks.bridge.play.controllers.CacheAdminController.listItems
 DELETE /v3/cache/:cacheKey @org.sagebionetworks.bridge.play.controllers.CacheAdminController.removeItem(cacheKey: String)
 
+# FOOTBALL PLAYERS HEATH STUDY API --------------------------------------------------------------------------
+
+GET    /fphs/externalId   @org.sagebionetworks.bridge.play.controllers.FPHSController.verifyExternalIdentifier(externalId: String ?= null)
```

Comment:
Eric Sieg had the opposite opinion (he wanted GET) and I didn't feel strongly about it, so I switched it from a POST. It's idempotent so it's okay to be a GET.